docs/language/operators/from.md

```diff
@@ -54,20 +66,105 @@ from ... | switch color (
 ) | ...
 ```
 
-The output of a fork consists of multiple legs that must be merged.
-If the downstream operator expects a single input, then the output legs are
-merged with an automatically inserted [combine operator](combine.md).
+### Input Data
+
+Examples below below assume the existence of the Zed lake created and populated
+by the following commands:
+
+```mdtest-command
+export ZED_LAKE=example
+zed -q init
+zed -q create coinflips
+echo '{flip:1,result:"heads"} {flip:2,result:"tails"}' | zed -q -use coinflips load -
+zed -q -use coinflips branch trial 
+echo '{flip:3,result:"heads"}' | zed -q -use coinflips@trial load -
+zed -q create numbers
+echo '{number:1,word:"one"} {number:2,word:"two"} {number:3,word:"three"}' | zed -q -use numbers load -
+zed ls | awk '{ print $1 }'
+```
+
+The lake then contains the two pools:
+
+```mdtest-output
+coinflips
+numbers
+```
+
+The following file `hello.zson` is also used.
+
+```mdtest-input hello.zson
+{greeting:"hello world!"}
+```
 
 ### Examples
 
-_Copy input to two paths and merge_
+_Source structured data from a local file_
+
+```mdtest-command
+zq -z 'file hello.zson | yield greeting'
+```
+=>
+```mdtest-output
+"hello world!"
+```
+
+_Source data from a local file, but in line format_
+```mdtest-command
+zq -z 'file hello.zson format line | head 1'
+```
+=>
+```mdtest-output
+"{greeting:\"hello world!\"}"
+```
+
+_Source structured data from a URI_
+```mdtest-command
+zq -z 'get https://raw.githubusercontent.com/brimdata/zui-insiders/main/package.json | yield productName'
+```
```

Comment:
I'm prepared to live without protecting this as an `mdtest-command`, but am submitting for consideration. The down side of protecting it is that if you're running the tests when not Internet-connected, the test will always fail. I'm not sure if the negatives of that outweigh the benefits of having a protected test. 🤷‍♂️